gadgets/trace_exec/program.bpf.c

```diff
@@ -38,6 +38,7 @@ struct event {
 	bool pupper_layer;
 	unsigned int args_size;
 	char comm[TASK_COMM_LEN];
+	char pcomm[TASK_COMM_LEN];
```

Comment:
Did you remove the new field `pcomm` in `gadgets/trace_exec/gadget.yaml` by mistake?